src/scancode/cli.py

```diff
@@ -785,4 +776,4 @@ def save_results(scanners, only_findings, files_count, results, format, options,
                     raise e
         return
 
-    write_formatted_output(scanners, files_count, version, notice, results, format, options, input, output_file, echo_stderr)
+    write_formatted_output(files_count, version, notice, results, format, options, input, output_file, echo_stderr, scan_output_plugins)
```

Comment:
It's the `PluginManager` instance initialized with the `scan_output` marker in `cli.py`, it is used to call the appropriate format plugin. I think I can get rid of it by supplying the appropriate plugin directly as an argument, is that a good idea?